src/components/FruitCounter.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+function FruitCounter ({fruit, state}) {
+
+    const [counter, setCounter] = React.useState(0)
+
+    if (state) {
+        setCounter(0)
+    }
+
+    function handleMinusClick() {
+        if (counter < 1) {
+            return setCounter(0)
+        }
+        else {
+            return setCounter(counter - 1)
+        }
```

Comment:
Ik had deze toch net wat anders ingestoken. Nu is het zo dat er altijd naar -1 gegaan kan worden en dan vanzelf weer op 0 gezet wordt. Dus twee rerenders teveel. Je had het ook zo kunnen doen, en dan ben je van de onnodige rerenders af:
```
if (counter > 0){
    setCounter(counter - 1);
}
```